app/controllers/mass_rollouts_controller.rb

```diff
@@ -47,34 +47,41 @@ def review_deploy
     end
   end
 
-  # TODO: show when deploys are waiting for buddy
```

Comment:
this is still a todo isn't it?

Reply:
decided to not do that since we have limited room and `no buddy` is visible in the deploy list we redirect to, so it's not that urgent